web/CustomExtensions/UserExtensions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Security.Claims;
+
+namespace BricksAndHearts.CustomExtensions
+{
+    public static class UserExtensions
+    {
+        public static bool IsLandlordOrAdmin(this ClaimsPrincipal user)
+        {
+            return user.IsInRole("Landlord") || user.IsInRole("Admin");
+        }
+    }   
+}
```

Comment:
We can use file scoping as we do elsewhere :) 
```suggestionusing System.Security.Claims;
namespace BricksAndHearts.CustomExtensions;

public static class UserExtensions
{
    public static bool IsLandlordOrAdmin(this ClaimsPrincipal user)
    {
        return user.IsInRole("Landlord") || user.IsInRole("Admin");
    }
}

```